### PR TITLE
New monitor kiosk sessions using RUM guide

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do?
Adds a new guide about using Datadog's SDK to monitor kiosk sessions with RUM

### Motivation
[DOCS-5766](https://datadoghq.atlassian.net/browse/DOCS-5766)

### Additional Notes
Ready to merge following a review by @nhientrandatadog, rum-browser, and rum-mobile

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5766]: https://datadoghq.atlassian.net/browse/DOCS-5766?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by nhientrandatadog — APPROVED** [review]
looks good to me!

File: content/en/real_user_monitoring/guide/monitor-kiosk-sessions-using-rum.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  text: 'RUM Dashboards'
+
+---
+
+## Overview
+Kiosk applications, such as fast food order machines and airline check-in terminals, often serve multiple users in quick succession. For that reason, triggering the end of a session based on a user action instead of waiting for an automatic session expiration (such as 15 minutes of inactivity or 4 hours total duration) is vital for collecting accurate session data and metrics for each user. With Datadog RUM SDKs, you can use this capability for an improved session tracking experience.
+
+## Use `stopSession()` when users end their interaction
+
+Use the SDK `stopSession()` method to stop the session when the user finishes their interaction with the application, like when returning to the home screen or logging out, for example. A new session is created as soon as a user interacts with the application again or when a new View is started (mobile only).
```
Comment (urseberry) [inline content/en/real_user_monitoring/guide/monitor-kiosk-sessions-using-rum.md:17]:
```suggestion
Use the SDK `stopSession()` method to stop the session when the user finishes their interaction with the application, like when returning to the home screen or logging out. A new session is created as soon as a user interacts with the application again or when a new View is started (mobile only).
```

File: content/en/real_user_monitoring/guide/monitor-kiosk-sessions-using-rum.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+If a user was identified within the session, you might want to clear user information after calling `stopSession()` to start afresh. See documentation based on your application's framework: [Browser][1], [iOS][2], [Android][3], [Flutter][4], [React Native][5].
```
Comment (urseberry) [inline content/en/real_user_monitoring/guide/monitor-kiosk-sessions-using-rum.md:19]:
```suggestion
If a user is identified within the session, you may want to clear user information after calling `stopSession()` to start afresh. See documentation based on your application's framework: [Browser][1], [iOS][2], [Android][3], [Flutter][4], [React Native][5].
```

**Review by urseberry — COMMENTED** [review]
Short and sweet. Love it. Just a couple of nits.

**Review by plousada — APPROVED** [review]

**Review by xgouchet — APPROVED** [review]

**Review by BenoitZugmeyer — APPROVED** [review]

**Pull request merged**